demos/shapes-custom-question-authoring-demo/src/question/index.js

```diff
@@ -0,0 +1,339 @@
+                if(kind === "triangle") {
+                    shapeBtn.setAttribute('style', `width:${pxValue+10}px;background:${color};`)
+                }
+                if(kind === "square" || kind === "circle") {
+                    shapeBtn.setAttribute('style', `width: ${pxValue}px; height: ${pxValue}px; background: ${color};`)
+                }
+                if(kind === "hexagon") {
+                    shapeBtn.setAttribute('style', `height: ${pxValue}px; background: ${color};`)
+                }
+                shapeWapper.appendChild(shapeBtn);
+                if(question.display_shape_names) {
+                    const p = document.createElement('p');
+                    p.innerText = kind;
+                    shapeWapper.appendChild(p)
+                }
+                el.querySelector('.shapes').appendChild(shapeWapper)
+            })
+            // if the question does not have any shapes defined, 
+            // then render a message to please add shapes first
+            if(!question.shapes) {
+                el.querySelector('.shapes').innerHTML = `<h3>Please add shapes first.</h3>`
+            }
+        });
+    }
+
+    /**
+     * Add public methods to the created question instance that is accessible during runtime
+     *
+     * Example: questionsApp.question('my-custom-question-response-id').myNewMethod();
+     * 
+     */
+    registerPublicMethods() {
+        const { init, el } = this;
+        // Attach the methods you want on this object
+        const facade = init.getFacade();
+
+        facade.disable = () => {
+            // TODO: Requires implementation
+            /**
+             * The purpose of this method is to prevent learner interaction with your question's UI.
+             * 
+             * If you plan to display your custom question in "review" state, then you need to implement this
+             * method to prevent a learner or instructor who is reviewing their completed results from being able to change the responses in your question UI.
+             */
+
+            // EXAMPLE implementation
+            // document.getElementById('my-input').setAttribute('disabled', true)
+            const shapeButtons = Array.from(el.querySelectorAll('.shape-wrapper button'));
+            shapeButtons.forEach(btn => btn.setAttribute('disabled', true));
+
+
+        };
+        facade.enable = () => {
+            /**
+             * The purpose of this method is to re-enable learner interaction with your question's UI
+             * after it has been previously disabled.
+             *               
+             * (For example, you plan to temporarily disable the question UI for a student taking the assessment until they complete another task like spend a set time reading the instructions.)
+             */
+
+            const shapeButtons = Array.from(el.querySelectorAll('.shape-wrapper button'));
+            shapeButtons.forEach(btn => btn.removeAttribute('disabled'));
```

Comment:
add line after const
```suggestion
            const shapeButtons = Array.from(el.querySelectorAll('.shape-wrapper button'));

            shapeButtons.forEach(btn => btn.removeAttribute('disabled'));
```